src/Pos/Launcher/Runner.hs

```diff
@@ -344,14 +345,14 @@ setupLoggers params = setupLogging =<< getRealLoggerConfig params
 loggerBracket :: LoggingParams -> IO a -> IO a
 loggerBracket lp = bracket_ (setupLoggers lp) releaseAllHandlers
 
+-- TODO: remove this function and all it's calls?
 addDevListeners
     :: MinWorkMode m => Timestamp
     -> ListenersWithOut m
     -> ListenersWithOut m
 addDevListeners sysStart ls =
     if Const.isDevelopment
-    then mergeLs [ stubListenerOneMsg (Proxy :: Proxy SysStartResponse)
-                 , sysStartReqListener sysStart] `mappendPair` ls
+    then mergeLs [ stubListenerOneMsg (Proxy :: Proxy MsgGetHeaders)] `mappendPair` ls
```

Comment:
@gromakovsky @volhovM I removed `SysStartResponse` data type. So I can use it here. I don't want to remove this function yet because removal of it will also remove a lot of functions. I need to put something that is `Message` after `Proxy` for this code to compile. Obviously, this should be removed. I just asking: is there any sense in old function. Should I remove it as whole or not?